projects/core/components/primitive-textfield/primitive-textfield.template.html

```diff
@@ -1,3 +1,4 @@
+<ng-container *ngIf="content?.changes | async"></ng-container>
```

Comment:
what do you think about add tests for this case?

Reply:
I don't think it's necessary